packages/conf-libevent/conf-libevent.1/opam

```diff
@@ -5,10 +5,24 @@ bug-reports: "https://github.com/ocaml/opam-repository/issues"
 authors: "Libevent dev team"
 license: "BSD-3-clause"
 build: [
-  ["pkg-config" "--exists" "libevent"]
+  "pkgconf" {os = "win32" & os-distribution != "cygwinports"}
+  "--personality=i686-w64-mingw32"
+    {os = "win32" & os-distribution != "cygwinports" &
+     host-arch-x86_32:installed}
+  "--personality=x86_64-w64-mingw32"
+    {os = "win32" & os-distribution != "cygwinports" &
+     host-arch-x86_64:installed}
+  "pkg-config" {os != "Win32" | os-distribution != "cygwin"}
+  "libevent"
```

Comment:
this dropped (on the non-win32 or cygwin) the `--exists` flag between `pkg-config` and `libevent` -- was this intentional?

Reply:
From the man page, `--exists` is implied when no other option is specified.

Reply:
@mtelvers Thanks for your comment. Since there are at least three pkg-config implementations, I'm not sure which one you're talking about. I find it very brittle to rely on the documented default of one of the pkg-config implementations.

Reply:
FWIW: http://pkg-config.freedesktop.org/ https://github.com/pkgconf/pkgconf https://metacpan.org/release/PkgConfig